packages/pwa-kit-runtime/src/utils/ssr-namespace-paths.js

```diff
@@ -5,39 +5,53 @@
  * For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
  */
 
+import {getConfig} from './ssr-config'
+
 /**
  * This file defines the /mobify paths used to set up our Express endpoints.
  *
  * If a namespace for the /mobify paths is defined, the methods in here will return the
  * namespaced path. ie. /namespace/mobify/...
  */
 
+// The MOBIFY_PATH is defined separately in preparation for the future eventual removal or
+// replacement of the 'mobify' part of these paths
 const MOBIFY_PATH = '/mobify'
 const PROXY_PATH_BASE = `${MOBIFY_PATH}/proxy`
 const BUNDLE_PATH_BASE = `${MOBIFY_PATH}/bundle`
 const CACHING_PATH_BASE = `${MOBIFY_PATH}/caching`
 const HEALTHCHECK_PATH = `${MOBIFY_PATH}/ping`
 const SLAS_PRIVATE_CLIENT_PROXY_PATH = `${MOBIFY_PATH}/slas/private`
 
-/**
- * @private
- */
-const _getNamespace = () => {
-    // TODO - namespaces for /mobify path will be implemented at a later date.
-    // Returns an empty string for now.
-    // Below is an example of what this implementation might look like.
-    /*
-        let {namespace = ""} = getConfig()
-        namespace = typeof namespace === 'function' ? namespace() : namespace
-        return namespace
-    */
-    return ''
+export const getEnvBasePath = () => {
+    // getConfig is memoized on the server so we are not needing to make
+    // multiple reads of the config file
+    // on the client, this will initially not return anything as
+    // window.__config__ first needs to be hydrated
+    // we cannot memoize getEnvBasePath as it's value may change once
+    // window.__config__ is hydrated
+    const config = getConfig()
+    let basePath = config?.envBasePath ? config.envBasePath : ''
+
+    if (typeof basePath !== 'string') {
+        console.log('Invalid environment base path configuration. Using default base path.')
```

Comment:
Let's use the PWAKitLogger instead of `console` object.

```suggestion
        logger.warn('Invalid environment base path configuration. Using default base path.')
```

Reply:
fixed